migrations/20201125223742-add-pin-type-and-parent-id.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.addColumn(
+          'locationPins',
+          'pinType',
+          {
+            type: Sequelize.DataTypes.ENUM('EVENT', 'CCP', 'OTHER'),
+          },
+          { transaction: t }
+        ),
+        queryInterface.addColumn(
+          'locationPins',
+          'ccpParentId',
+          {
+            type: Sequelize.INTEGER,
+          },
+          { transaction: t }
+        ),
+      ]).then(() => {
+        queryInterface.sequelize.query(
+          `UPDATE "locationPins" SET "pinType" = 'OTHER' WHERE "pinType" IS NULL`
+        );
+      });
```

Comment:
minor nit - also i would double check the formatting (run it through prettier)
```suggestion
      ]).then(() =>
        queryInterface.sequelize.query(
          `UPDATE "locationPins" SET "pinType" = 'OTHER' WHERE "pinType" IS NULL`
        );
     );
```